### PR TITLE
feat(BUX-121): library absorbtion - initializes repository with current go-paymail version

### PR DESCRIPTION
**Pull request by wregulski**

### CODE REVIEW

File: _.github/SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+## Supported & Maintained Versions
+
+| Version | Supported          |
+|---------|--------------------|
+| 0.x.x   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+Individuals or organizations that are experiencing a product security issue are strongly encouraged to contact the [project maintainers](mailto:security@tonicpow.com).
```
Comment (arkadiuszos4chain) [inline _.github/SECURITY.md:11]: Is it valid mail?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+include .make/go.mk
+
+## Not defined? Use default repo name which is the application
+ifeq ($(REPO_NAME),)
+	REPO_NAME="go-paymail"
+endif
+
+## Not defined? Use default repo owner
+ifeq ($(REPO_OWNER),)
+	REPO_OWNER="tonicpow"
```
Comment (pawellewandowski98) [inline Makefile:14]: Should we change it to bitcoin-sv?

File: brfc_definintions.go [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+	BRFCPayToProtocolPrefix            = "7bd25e5a1fc6"       // more info: http://bsvalias.org/04-04-payto-protocol-prefix.html
+	BRFCPki                            = "pki"                // more info: http://bsvalias.org/03-public-key-infrastructure.html
+	BRFCPkiAlternate                   = "0c4339ef99c2"       // more info: http://bsvalias.org/03-public-key-infrastructure.html
+	BRFCPublicProfile                  = "f12f968c92d6"       // more info: https://github.com/bitcoin-sv-specs/brfc-paymail/pull/7/files
+	BRFCReceiverApprovals              = "3d7c2ca83a46"       // more info: http://bsvalias.org/04-03-receiver-approvals.html
+	BRFCSenderValidation               = "6745385c3fc0"       // more info: http://bsvalias.org/04-02-sender-validation.html
+	BRFCSFPAssetInformation            = "1300361cb2d4"       // more info: https://docs.moneybutton.com/docs/paymail/paymail-08-asset-information.html
+	BRFCSFPAuthoriseAction             = "95dddb461bff"       // more info: https://docs.moneybutton.com/docs/sfp/paymail-10-sfp-authorise.html
+	BRFCSFPBuildAction                 = "189e32d93d28"       // more info: https://docs.moneybutton.com/docs/sfp/paymail-09-sfp-build.html
```
Comment (kuba-4chain) [inline brfc_definintions.go:18]: Moneybutton docs don't work :(

File: Makefile (hunk 2 above)
Comment (arkadiuszos4chain) [reply Makefile:14]: I believe this will be handled in https://bitcoinassociation.atlassian.net/browse/BUX-142

File: brfc_definintions.go [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+	BRFCVerifyPublicKeyOwner           = "a9f510c16bde"       // more info: http://bsvalias.org/05-verify-public-key-owner.html
+)
+
+// BRFCKnownSpecifications is a running list of all known BRFC specifications
+//
+// JSON file was converted into a go:var for binary shipment
+// Add your spec at runtime using LoadBRFCs()
+// Add your spec to the list below: https://github.com/tonicpow/go-paymail/issues/new/choose
```
Comment (kuba-4chain) [inline brfc_definintions.go:26]: Should we change to  `bitcoinsv`?

File: brfc_definintions.go [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+   "id": "a9f510c16bde",
+   "title": "bsvalias public key verify (Verify Public Key Owner)",
+   "url": "http://bsvalias.org/05-verify-public-key-owner.html",
+   "version": "1"
+  },
+  {
+   "author": "Ryan X. Charles (Money Button), Miguel Duarte (Money Button), Rafa Jimenez Seibane (Handcash), Ivan Mlinarić  (Handcash)",
+   "id": "5f1323cddf31",
+   "title": "P2P Transactions",
+   "url": "https://docs.moneybutton.com/docs/paymail-06-p2p-transactions.html",
```
Comment (kuba-4chain) [inline brfc_definintions.go:84]: Link doesn't work

File: brfc_definintions.go [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+   "version": "1"
+  },
+  {
+   "author": "Ryan X. Charles (Money Button), Miguel Duarte (Money Button), Rafa Jimenez Seibane (Handcash), Ivan Mlinarić  (Handcash)",
+   "id": "2a40af698840",
+   "title": "P2P Payment Destination",
+   "url": "https://docs.moneybutton.com/docs/paymail-07-p2p-payment-destination.html",
```
Comment (kuba-4chain) [inline brfc_definintions.go:91]: Same here

File: brfc_definintions.go (hunk 4 above)
Comment (pawellewandowski98) [inline brfc_definintions.go:26]: should be changed to bitcoin-sv/go-paymail

File: brfc_definintions.go [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+   "id": "a0a4c8b96133",
+   "title": "spv_channels",
+   "url": "https://github.com/bitcoin-sv-specs/brfc-spvchannels",
+   "version": "1.0.0-beta"
+  },
+  {
+   "author": "Fabriik",
+   "id": "1300361cb2d4",
+   "title": "Asset Information",
+   "url": "https://docs.moneybutton.com/docs/paymail/paymail-08-asset-information.html",
```
Comment (kuba-4chain) [inline brfc_definintions.go:168]: Same here

File: brfc_definintions.go [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+   "version": "1"
+  },
+  {
+   "author": "Fabriik",
+   "id": "189e32d93d28",
+   "title": "Simple Fabriik Protocol for Tokens Build Action",
+   "url": "https://docs.moneybutton.com/docs/sfp/paymail-09-sfp-build.html",
```
Comment (kuba-4chain) [inline brfc_definintions.go:175]: And here

File: brfc_definintions.go [hunk 9]
```diff
@@ -0,0 +1,193 @@
...
+   "id": "95dddb461bff",
+   "title": "Simple Fabriik Protocol for Tokens Authorise Action",
+   "url": "https://docs.moneybutton.com/docs/sfp/paymail-10-sfp-authorise.html",
+   "version": "1"
+  },
+  {
+   "author": "Fabriik",
+   "id": "f792b6eff07a",
+   "title": "P2P Payment Destination with Tokens Support",
+   "url": "https://docs.moneybutton.com/docs/paymail/paymail-11-p2p-payment-destination-tokens.html",
```
Comment (kuba-4chain) [inline brfc_definintions.go:189]: And here

File: examples/client/get_public_profile/get_public_profile.go [hunk 10]
```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"log"
+
+	"github.com/tonicpow/go-paymail"
```
Comment (kuba-4chain) [inline examples/client/get_public_profile/get_public_profile.go:6]: Should we change to `bitcoin-sv`?

Here and in many other situations?

File: Makefile (hunk 2 above)
Comment (pawellewandowski98) [reply Makefile:14]: I think that this task is to change other repos which are using go-paymail from tonicpow organization

**Pull request closed without merging**